cognite/client/data_classes/contextualization.py

```diff
@@ -214,15 +214,19 @@ def refit(self, true_matches: List[Union[Dict, Tuple[Union[int, str], Union[int,
         )
         return self._load(response.json(), cognite_client=self._cognite_client)
 
+    @staticmethod
+    def _flatten_entity(entity: Dict) -> Dict:
+        if isinstance(entity, CogniteResource):
+            entity = entity.dump(camel_case=True)
+        if "metadata" in entity:
+            for k, v in entity["metadata"].items():
+                entity["metadata.{}".format(k)] = v
```

Comment:
should there be a `del entity["metadata"]` afterwards here?

Reply:
the dict comprehension takes care of this